website/content/labs/lab5.md

```diff
@@ -116,7 +114,9 @@ It’s time to write your first tests! In your `test_segment.c` file, do the fol
 
 * Write at least one test case for each of `segment_new`, `segment_init`, `segment_free`.  You may find it helpful to look at similar tests in `test_point.c` and `test_polygon.c`.
 * We already had some tests for `segment_intersect` in `test_point.c`. *Refactor* them into `test_segment.c`, and make sure they're in the correct test suite!
-* Write test cases for `on_segment` and `orientation`. Remember that, in Lab 2, you had the option of moving these functions to `segment.c`, or to keep them in `point.c` (and exposing them through `point.h`). If you moved them to `segment.c`, add the tests to `test_segment.c`; otherwise, add them to `test_point.c`. For these test cases, remember that you should have as much *coverage* as possible: your test cases should cover as many outcomes (and as many flows of execution through the individual function) as possible.
+* Write test cases for `on_segment` and `point_orientation` (previously known as `orientation`). For these test cases, remember that you should have as much *coverage* as possible: your test cases should cover as many outcomes (and as many flows of execution through the individual function) as possible.
```

Comment:
Is it possible to give them an estimate of how many tests we might expect to see? I feel like this might cause some students to feel like they have to write a ton of tests

Reply:
I've clarified that we're going to be focusing on coverage, not on the number of tests, but that it should be possible to provide sufficient coverage with 3-4 tests for each function.